### PR TITLE
3848 - Move com.ibm.was.jaxrs.2.0.cdi_fat to OpenLiberty

### PR DESCRIPTION
**Pull request by WhiteCat22**
https://github.com/OpenLiberty/open-liberty/issues/3848

### CODE REVIEW

**Comment by WhiteCat22** [comment]
Tests are all running, all legal headers are added and code has been cleaned up. May need to still clean up the bnd file and .classpath files etc...

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

**Comment by WhiteCat22** [comment]
#build

File: dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.settings/org.eclipse.jdt.ui.prefs [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
...strumentation and @FFDCIgnore\n// http\://was.pok.ibm.com/xwiki/bin/view/Liberty/LoggingFFDC\n${exception_var}.printStackTrace();</template><template autoinsert\="true" context\="methodbody_context" deleted\="false" description\="Code in created method stubs" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.methodbody" name\="methodbody">// ${todo} Auto-generated method stub\n${body_statement}</template><template autoinsert\="true" context\="constructorbody_context" deleted\="false" description\="Code in created constructor stubs" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.constructorbody" name\="constructorbody">${body_statement}\n// ${todo} Auto-generated constructor stub</template><template autoinsert\="true" context\="getterbody_context" deleted\="false" description\="Code in created getters" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.getterbody" name\="getterbody">return ${field};</template><template autoinsert\="true" context\="setterbody_context" deleted\="false" description\="Code in created setters" enabled\="true" id\="org.eclipse.jdt.ui.text.codetemplates.setterbody" name\="setterbody">${field} \= ${param};</template></templates>
```
Comment (aguibert) [inline dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.settings/org.eclipse.jdt.ui.prefs]: the header setting have the old IBM OCO copyright in them

File: dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.classpath [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```
Comment (aguibert) [inline dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.classpath]: this .classpath won't work in OpenLiberty.  Check build.example_fat/.classpath for an example

File: dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.project [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
```
Comment (aguibert) [inline dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.project]: not required, but should have the bndtools nature on the .project file (see example FAT for template)

**Comment by WhiteCat22** [comment]
#build

File: dev/log.log [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Support for running Gradle using Java 7 has been deprecated and is scheduled to be removed in Gradle 5.0. Please see https://docs.gradle.org/4.3.1/userguide/java_plugin.html#sec:java_cross_compilation for more details.
```
[the remark this answers is not in the document]
Comment (WhiteCat22) [reply dev/log.log]: wow good catch, got added by mistake when I was making Andy's changes

File: dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.classpath (hunk 2 above)
Comment (WhiteCat22) [reply dev/com.ibm.ws.jaxrs.2.0.cdi.1.2_fat/.classpath]: updated

**Pull request merged**